os/binfmt/libelf/Kconfig

```diff
@@ -48,3 +48,10 @@ config ELF_EXIDX_SECTNAME
 		be loaded by the ELF binary loader.
 
 		This is needed to support exception handling on loadable ELF modules.
+
+config ELF_INCLUDE_SYMBOLS
```

Comment:
1. How about `exclude` and `ifeq ($(CONFIG_ELF_EXCLUDE_SYMBOLS),y)`?
`include` looks positive but real operation `ifneq` is negative.
I think exclude is better to understand.
2.  Is this proper location to add? Here is not a location of making binary. Isn't it?
3. Symbol information is necessary to debug using T32. Then, how about making a new binary which is stripped?
I mean after making non-stripped elf and then making new stripped file to download.

Reply:
for 3rd, Cypress does like what I said.